sentiment_analysis_model.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+"""
+
+import numpy as np 
+import pandas as pd 
+from sklearn.model_selection import train_test_split 
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.classify import SklearnClassifier
+
+from wordcloud import wordcloud,STOPWORDS
+
+from subprocess import check_output
+
+import mlflow 
+
+
+class Sentiment:
+   def __init__(self):
+    
+       data = pd.read_csv(r'C:\Users\user\Downloads\Sentiment.csv')

```

Comment:
1. Please download csv on run time or add description on how to obtain it. what should be the structure.
2. Please do not hard code path. 
   - Try to have all necessary files within current directory or sub directories. Refer following coda as an example.
     [definition](https://github.com/produvia/ai-platform/blob/c5962786cb999c5adee95b0d11c423f0766b9a63/tasks/computer-vision/object-detection/download_file.py), [declaration](https://github.com/produvia/ai-platform/blob/c5962786cb999c5adee95b0d11c423f0766b9a63/tasks/computer-vision/object-detection/yolo3_weights_to_keras.py#L4) and [call](https://github.com/produvia/ai-platform/blob/c5962786cb999c5adee95b0d11c423f0766b9a63/tasks/computer-vision/object-detection/yolo3_weights_to_keras.py#L167)
   - Use python modules for keeping code cross platform
   - If file is present in current directory then following code should work. Similarly If file is present in sub directory then you could add sub directory separated by comma in between cwd and 'Sentiment.csv'.
     ```
     import os
     cwd = os.getcwd()
     data = pd.read_csv(os.path.join(cwd, 'Sentiment.csv'))
     ```